.babelrc

```diff
@@ -1,14 +1,19 @@
 {
     "env": {
-        "production": {
-            "presets": [
-                "latest"
-            ]
-        },
+        "production": {},
         "development": {
             "plugins": [
                 "transform-async-to-generator"
             ]
+        },
+        "admin": {
+            "presets": [
+                ["env", {
+                    "targets": {
+                        "browsers": ["last 2 Chrome versions, last 1 Safari version, last 2 Firefox versions, last 2 Edge versions"]
```

Comment:
Does this go back far enough? Does this mean that once we start using ES6 <=IE11 will go boom?